diff --git a/node_modules/react-native-snap-carousel/src/utils/animations.ts b/node_modules/react-native-snap-carousel/src/utils/animations.ts
index bf1c50f..a438f1a 100644
--- a/node_modules/react-native-snap-carousel/src/utils/animations.ts
+++ b/node_modules/react-native-snap-carousel/src/utils/animations.ts
@@ -1,7 +1,7 @@
 import { Platform, Animated } from 'react-native';
 import type { CarouselProps } from 'src/carousel/types';
 
-const IS_ANDROID = Platform.OS === 'android';
+const IS_ANDROID = false;
 
 // Get scroll interpolator's input range from an array of slide indexes
 // Indexes are relative to the current active slide (index 0)
